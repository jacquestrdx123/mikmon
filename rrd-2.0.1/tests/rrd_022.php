<?php
var_dump(rrdc_disconnect());
?>
