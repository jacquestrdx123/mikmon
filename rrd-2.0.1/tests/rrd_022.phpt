--TEST--
rrdc_disconnect test
--SKIPIF--
<?php
include('skipif.inc');
if (!function_exists("rrdc_disconnect")) {
	die("skip rrdc_disconnect only in rrdtool >= 1.4");
}

?>
--FILE--
<?php
var_dump(rrdc_disconnect());
?>
--EXPECTF--
NULL
