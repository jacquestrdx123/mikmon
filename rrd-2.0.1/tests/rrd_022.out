Warning: Missing arginfo for rrd_error() in Unknown on line 0

Warning: Missing arginfo for rrdc_disconnect() in Unknown on line 0

Warning: Missing arginfo for rrd_version() in Unknown on line 0

Warning: Missing arginfo for RRDGraph::save() in Unknown on line 0

Warning: Missing arginfo for RRDGraph::saveVerbose() in Unknown on line 0

Warning: Missing arginfo for RRDCreator::save() in Unknown on line 0
NULL